### PR TITLE
osbuild/images 0.55

### PR DESCRIPTION
**Pull request by croissanne**
TODO:

- [x] push regression test fix
- [x] cloud-image-val: https://github.com/osbuild/cloud-image-val/pull/311
- [x] push new cloud-image-val to :prod tag
- [x] fix koji container: https://github.com/osbuild/containers/pull/71
- [x] push new koji container: https://github.com/osbuild/containers/actions/runs/8706131081
- [x] cloud-image-val centos-9 NetworkManager selinux denials skip: https://github.com/osbuild/cloud-image-val/pull/312
- [x] push new cloud-image-val to :prod

### CODE REVIEW

File: internal/cloudapi/v2/server.go [hunk 1]
```diff
@@ -591,7 +591,7 @@ func serializeManifest(ctx context.Context, manifestSource *manifest.Manifest, w
 		}
 	}
 
-	ms, err := manifestSource.Serialize(depsolveResults.PackageSpecs, containerSpecs, ostreeCommitSpecs, despolveResults.RepoConfigs)
+	ms, err := manifestSource.Serialize(depsolveResults.PackageSpecs, containerSpecs, ostreeCommitSpecs, depsolveResults.RepoConfigs)
```
Comment (achilleas-k) [inline internal/cloudapi/v2/server.go:594]: This typo fix should be squashed into the previous commit where it's introduced.

**Review by achilleas-k — COMMENTED** [review]
LGTM but I have a tiny nitpick related to git history :)

File: internal/cloudapi/v2/server.go (hunk 1 above)
Comment (croissanne) [reply internal/cloudapi/v2/server.go:594]: oh yes, srry must've missed that

**Review by achilleas-k — DISMISSED** [review]

**Comment by supakeen** [comment]
This is also tracked for Fedora IoT at: https://pagure.io/fedora-iot/issue/57

**Review by achilleas-k — APPROVED** [review]

**Review by bcl — APPROVED** [review]
Looks good! I especially like having a separate commit for all the places where the new repoconfigs are ignored.

**Pull request merged**